code/modules/projectiles/projectile.dm

```diff
@@ -289,7 +289,7 @@
 	GLOB.round_statistics.total_projectiles_fired++
 	SSblackbox.record_feedback("tally", "round_statistics", 1, "total_projectiles_fired")
 
-	if(ammo.flags_ammo_behavior & AMMO_BALLISTIC)
+	if(ammo.flags_ammo_behavior & AMMO_BALLISTIC && !recursivity)
 		GLOB.round_statistics.total_bullets_fired++
```

Comment:
?

Reply:
So we don't count twice bullets that passed through a portal